src/routes/index.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+
+import AuthRouter from './auth/auth.router';
+import HealthRouter from './health/health.router';
+import StorageRouter from './storage/storage.router';
+
+const Routes = (): Router => {
+  const router = Router();
+
+  router.use('/auth', AuthRouter);
+  router.use('/health', HealthRouter);
```

Comment:
this route should be relocated to the root level (i.e. at `/`, not in `/api`), as this is meant to indicate the server's own health, regardless of the different endpoints' statuses.

https://github.com/supercluster-covid-data-portal/infra/blob/main/helm/api/dev/values.yaml#L62